### PR TITLE
Refactor compute methods calls

### PR DESCRIPTION
**Pull request by bogdanfazakas**
Fixes # .

Changes proposed in this PR:

- bump oceanjs lib to 0.14.0
- updated compute methods calls after refactor released with oceanJS 0.14.0

### CODE REVIEW

**Comment by bogdanfazakas** [comment]
blocked until we merge https://github.com/oceanprotocol/ocean.js/pull/731 and release a new version of oceanJS

**Review by mihaisc — APPROVED** [review]

**Pull request merged**
